bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>30.0-jre</version>
+      <version>30.1.1-jre</version>
```

Comment:
I dont think this is necessary

Reply:
here is the enforcer output when at 30.0-jre:
```
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for com.google.guava:guava:30.0-jre paths to dependency are:
+-com.google.cloud.bigtable:bigtable-hbase-1.x-tools:1.22.1-SNAPSHOT
  +-com.google.guava:guava:30.0-jre
and
+-com.google.cloud.bigtable:bigtable-hbase-1.x-tools:1.22.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x-shaded:1.22.1-SNAPSHOT
    +-io.grpc:grpc-census:1.38.0
      +-com.google.guava:guava:30.1-android
and
+-com.google.cloud.bigtable:bigtable-hbase-1.x-tools:1.22.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x-shaded:1.22.1-SNAPSHOT
    +-io.grpc:grpc-census:1.38.0
      +-io.grpc:grpc-api:1.38.0
        +-com.google.guava:guava:30.1-android
, 
Require upper bound dependencies error for com.google.errorprone:error_prone_annotations:2.3.4 paths to dependency are:
+-com.google.cloud.bigtable:bigtable-hbase-1.x-tools:1.22.1-SNAPSHOT
  +-com.google.guava:guava:30.0-jre
    +-com.google.errorprone:error_prone_annotations:2.3.4
and
+-com.google.cloud.bigtable:bigtable-hbase-1.x-tools:1.22.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x-shaded:1.22.1-SNAPSHOT
    +-io.grpc:grpc-census:1.38.0
      +-com.google.errorprone:error_prone_annotations:2.4.0
and
+-com.google.cloud.bigtable:bigtable-hbase-1.x-tools:1.22.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x-shaded:1.22.1-SNAPSHOT
    +-io.grpc:grpc-census:1.38.0
      +-io.grpc:grpc-api:1.38.0
        +-com.google.errorprone:error_prone_annotations:2.4.0
```

Reply:
in bigtable-hbase-1.x-tools/pom.xml, exclude grpc-census from bigtable-hbase-1.x-shaded. 

grpc-census gets shaded in bigtable-hbase-1.x-shaded and shouldnt leak here, but it does because of MNG-5899 & MSHADE-206

Reply:
thank you! Fixed!